### PR TITLE
ENH: use a single tempdir

### PR DESCRIPTION
**Pull request by UranusSeven**
## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass

### CODE REVIEW

**Comment by ChengjieLi28** [comment]
I tested this pr, and found that 
1. supervisor log and worker log are the same on the xorbits web ui:
image

image

2. on the web ui, supervisor log has so much info but the single file under /tmp/xorbits/log/supervisor_xxx has just two lines log.
image

I just open the Python Console and use this codes to start a local cluster:
```python
import xorbits.pandas as pd

pd.DataFrame({'a': [1, 2, 3, 4]}).mean()
```

Please check that whether these are the expected behavior.

**Comment by UranusSeven** [comment]
> I tested this pr, and found that
> 
> 1. supervisor log and worker log are the same on the xorbits web ui:
> 
> image
> 
> image
> 
> 2. on the web ui, supervisor log has so much info but the single file under /tmp/xorbits/log/supervisor_xxx has just two lines log.
> 
> image
> 
> I just open the Python Console and use this codes to start a local cluster:
> 
> ```python
> import xorbits.pandas as pd
> 
> pd.DataFrame({'a': [1, 2, 3, 4]}).mean()
> ```
> 
> Please check that whether these are the expected behavior.

Fixed. The issue is caused by MARS_LOG_DIR_KEY being set twice. And the value will always be worker's log dir since workers always get initialized after the supervisor. And because FileLoggerActor(s) take the env var MARS_LOG_DIR_KEY as the log dir, the content of supervisor and workers' logs are the same.

File: python/xorbits/_mars/deploy/oscar/pool.py [hunk 1]
```diff
@@ -184,12 +187,14 @@ def _get_log_config_path(log_config: Optional[str]) -> str:
...
     logging_conf: Dict[str, Any]
 ) -> Optional[configparser.RawConfigParser]:
-    parsed_logging_conf = _parse_file_logging_config(logging_conf)
+    parsed_logging_conf, log_config_path = _parse_file_logging_config(logging_conf)
 
     logging.config.fileConfig(
         parsed_logging_conf,
         disable_existing_loggers=False,
     )
+    logging.info("Logging configurations from %s have been applied", log_config_path)
```
Comment (ChengjieLi28) [inline python/xorbits/_mars/deploy/oscar/pool.py:196]: Should this be "logger.info" ?

File: python/xorbits/_mars/services/cluster/file_logger.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ class FileLoggerActor(mo.Actor):
     """
 
     def __init__(self):
+        super().__init__()
```
Comment (ChengjieLi28) [inline python/xorbits/_mars/services/cluster/file_logger.py:33]: What is this for?

Comment (UranusSeven) [reply python/xorbits/_mars/services/cluster/file_logger.py:33]: Sub class should always call the constructor of their parent class.

File: python/xorbits/_mars/deploy/oscar/pool.py [hunk 3]
```diff
@@ -186,12 +187,14 @@ def _get_log_config_path(log_config: Optional[str]) -> str:
...
```
Comment (UranusSeven) [inline python/xorbits/_mars/deploy/oscar/pool.py:196]:
```suggestion
    logger.info("Logging configurations from %s have been applied", log_config_path)
```

**Comment by ChengjieLi28** [comment]
> 



> workers



> Fixed. The issue is caused by MARS_LOG_DIR_KEY being set twice. And the value will always be worker's log dir since workers always get initialized after the supervisor. And because FileLoggerActor(s) take the env var MARS_LOG_DIR_KEY as the log dir, the content of supervisor and workers' logs are the same.

Tested. Xorbits log works well both in IPython and cmdline (python -m).

**Review by ChengjieLi28 — DISMISSED** [review]

**Review by aresnow1 — APPROVED** [review]
LGTM

**Review by ChengjieLi28 — APPROVED** [review]

**Pull request merged**